modules/govuk/manifests/apps/email_alert_api/sidekiq_queue_check.pp

```diff
@@ -0,0 +1,37 @@
+# == Define: govuk::apps::email_alert_api::sidekiq_queue_check
+#
+# Creates an Icinga check which monitors a sidekiq queue.
+#
+# === Parameters
+#
+# [*ensure*]
+#   Whether to enable the check.
+#   Default: present
+#
+# [*title*]
+#   The name of the sidekiq queues to look for.
+#
+# [*latency_warning*]
+#   The warning threshold for a sidekiq queue in minutes.
+#   Default: 5
+#
+# [*latency_critical*]
+#   The critical threshold for a sidekiq queue in minutes.
+#   Default: 10
+#
+define govuk::apps::email_alert_api::sidekiq_queue_check (
+  $ensure = present,
+  $latency_warning = undef,
+  $latency_critical = undef,
```

Comment:
As these `latency_...` parameters are required by `icinga::check::graphite`, I'd just make them mandatory (rather than defaulting to 'undef') if that's possible.